software/kallistobustools/count/main.nf

```diff
@@ -0,0 +1,58 @@
+// Import generic module functions
+include { initOptions; saveFiles; getSoftwareName } from './functions'
+
+params.options = [:]
+options        = initOptions(params.options)
+
+process KALLISTOBUSTOOLS_COUNT {
+    tag "$meta.id"
+    label 'process_medium'
+    publishDir "${params.outdir}",
+        mode: params.publish_dir_mode,
+        saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:getSoftwareName(task.process), publish_id:meta.id) }
+
+    conda (params.enable_conda ? "bioconda::kb-python=0.25.1" : null)
+    if (workflow.containerEngine == 'singularity' && !params.singularity_pull_docker_container) {
+        container "https://depot.galaxyproject.org/singularity/kb-python:0.25.1--py_0"
+    } else {
+        container "quay.io/biocontainers/kb-python:0.25.1--py_0"
+    }
+
+    input:
+    tuple   val(meta),  path(reads)
+    path    index
+    path    t2g
+    path    t1c
+    path    t2c
+    val     use_t1c
+    val     use_t2c
+    val     workflow
+    val     technology
+
+    output:
+    tuple val(meta), path "*_kallistobustools_count" , optional:false , emit: kallistobustools_count
+    tuple path "*.version.txt"             , optional:false , emit: version
```

Comment:
```suggestion
    tuple val(meta), path ("*_kallistobustools_count") , optional:false , emit: kallistobustools_count
    path "*.version.txt"             , optional:false , emit: version
```

Reply:
@FloWuenne this is what was causing the problem 🤦  it should run through like this

Reply:
Thank you for the fix @KevinMenden !!! Unfortunately, I am pretty occupied at work at the moment. Will probably only be able to test this early next week! Will try to wrap this module up and submit the references for the tests to test datasets as well then!